README.md

```diff
@@ -1,16 +1,82 @@
-# **Stock Market**
+# **Stock Market Project**
 
 ## Project summary
-In this project I created a system which simulates a stock market. The stock market composed of some companies. The system enables the user to insert and remove companies from the stock market and to make different actions.
+In this project I simulated stock market using two classes. The stock market class represents the entire set of companies that are found in the stock market and enables the user to conduct a variety of actions the different companies present in the market. By using the API given by the classes, different manipulations can be conducted such as changing net worth of companies, merging two companies and more. The company class represents a specific company and enables to edit parameters that charcterize the company. 
 
-##  About the Classes
-Two classes created to implement the system idea:
+##  Project Design
 
-1. *Company* - This class includes the required methods and represents a specific company.
+### Company class
+The company class is described using the following fields: 
+* **name** - represents the name of the company. The type of this field is string. 
+* **stocks_num** - represents the number of company's stocks. The type of this field is integer. 
+* **stock_price** - represents the price of one stock. The type of this field is integer or float.
+* **comp_type** - represents the type of the company. For example: High Tech, Pharmaceutical, Hardware, etc. The type of this field is string. 
 
-2. *StockMarket* - This class represents the whole stock market which contains a list of companies.
+The company class API includes the following functions:
+
+* *net_worth* - calculates the company's net worth. 
+
+* *set_name* - updates the name of the company
+
+* *set_stocks_num* - updates the number of company's stocks. 
+
+* *set_stocks_price* - updates the price of company's stock.
+
+* *set_comp_type* - updates the company type.
+
+* *update_net_worth* - updates the company's net worth such that number of stocks stay the same and the price of a stock changed. 
+
+* *add_stocks* - addes a number to the current number of stocks. 
+
+* *__repr__* - magic function which returns information about the company. 
+For example: 
+    
+        "NVIDIA Corporation, 1000 stocks, 
+         Price: 20.284, High Tech, Net Worth: 20284.0"
+
+* *__lt__*, *__gt__*, *__eq__* - operator overloading: 
+
+    lt = less than, gt = greate than, eq = equal to 
```

Comment:
Same Here, all information written here is known. A better documentation is explaining the way it is overloaded, for example: "When comparing between companies ( __eq__) the company with the greater net_value (i think?) will be seen as the bigger one" Now this is new information, we all know (all python users should know that __XXX__ are magic functions ( b.t.w they can also be called dunder methods, which is a short for "double underscore methods")